services/galley/src/Galley/Cassandra/Conversation/Members.hs

```diff
@@ -321,6 +321,15 @@ filterRemoteConvMembers users (qUntagged -> Qualified conv dom) =
         . retry x1
         $ query Cql.selectRemoteConvMembers (params LocalQuorum (user, dom, conv))
 
+lookupLocalMemberRemoteConv ::
+  UserId ->
+  Remote ConvId ->
+  Client [UserId]
+lookupLocalMemberRemoteConv userId (qUntagged -> Qualified conv dom) =
+  fmap (map runIdentity)
+    . retry x5
+    $ query Cql.selectRemoteConvMembers (params LocalQuorum (userId, dom, conv))
```

Comment:
This should probably use `query1`.

Reply:
Nice catch! Fix done in https://github.com/wireapp/wire-server/pull/2618/commits/69c7365cb38a7b6b23f7ebfa819c6967b2ebbc6f.